### PR TITLE
Issue #56: Adds vault-ui ingress

### PR DESCRIPTION
**Pull request by SonOfLope**
We want to enable access to the vault-ui to enable the team to create secrets.

TODO before ready for merge: 
~- [ ] Find fix to enable vault config operator to have enough permissions to access and configure vault.~

Preceding task will be done in another issue since its to provide automation of vault configuration but we have problems enabling the operator to have proper permissions on the cluster to interact with vault. Since this is not a priority we will open an issue.

### CODE REVIEW

**Review by rngadam — CHANGES_REQUESTED** [review]
nice diagram!

Some of the filenames have inconsistent cases when we expect lowercase everywhere. Maybe an additional check to add to repo standards?

**Comment by SonOfLope** [comment]
> nice diagram!
> 
> Some of the filenames have inconsistent cases when we expect lowercase everywhere. Maybe an additional check to add to repo standards?

Added issue to https://github.com/ai-cfia/github-workflows/issues/105

**Comment by rngadam** [comment]
> Updated file name

I still see an uppercase PNG extension

**Comment by SonOfLope** [comment]
> > Updated file name
> 
> I still see an uppercase PNG extension

done

**Review by ThomasCardin — APPROVED** [review]
LGTM

**Review by rngadam — APPROVED** [review]

File: kubernetes/aks/system/vault/base/vault-config-operator/policies.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  authentication:
+    path: kubernetes
+    role: config-admin
+  policy: |
+    # create secrets
+    path "kv/data/{{identity.entity.aliases.${auth/kubernetes/@accessor}.metadata.service_account_namespace}}" {
+      capabilities = [ "create", "update", "delete" ]
+    }
+---
+apiVersion: redhatcop.redhat.io/v1alpha1
```
[the remark this answers is not in the document]
Comment (SonOfLope) [reply kubernetes/aks/system/vault/base/vault-config-operator/policies.yaml:15]: referenced in https://github.com/ai-cfia/howard/issues/99

File: docs/secret-management.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# Secret management
```
[the remark this answers is not in the document]
Comment (SonOfLope) [reply docs/secret-management.md:1]: Opened https://github.com/ai-cfia/dev-rel-docs/issues/117

**Pull request merged**